website/templates/project/nodes_privacy.mako

```diff
@@ -1,4 +1,4 @@
-<div id="nodesPrivacy" class="modal fade" div style="display: none;">
```

Comment:
See line 3. This appeared to be a markup error, judging from the div attribute on the div tag.

I do recall verifying that the page looked the same for me when viewing the page directly.